.changeset/lovely-kiwis-juggle.md

```diff
@@ -0,0 +1,13 @@
+---
+'gatsby-plugin-clerk': patch
+'@clerk/clerk-sdk-node': patch
+'@clerk/nextjs': patch
+'@clerk/remix': patch
+'@clerk/clerk-expo': patch
+---
+
+Warn about environment variables deprecations:
```

Comment:
🙃 
```suggestion
Show warnings when the following deprecated environment variables are used: `CLERK_API_KEY`, `CLERK_FRONTEND_API`, `NEXT_PUBLIC_CLERK_FRONTEND_API`
```

Reply:
I will keep it as is for consistency to the previous deprecation warning changelog entries.